dex/version/version_buildinfo.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2013-2014 The btcsuite developers
+// Copyright (c) 2015-2022 The Decred developers
```

Comment:
I don't get these lines.  Surely this `vcsCommitID` didn't even exist until recently let alone back when it was btcsuite work.

Just 2021-2022 and the one author:
https://github.com/decred/dcrd/blob/master/internal/version/version_buildinfo.go